### DISCUSSION TITLE
要求退款

### DISCUSSION THREAD

**Discussion by Ginny125** [getlantern/lantern-forum-cn#1049] [General]
已在蓝灯中提交问题报告，这是ticket号{79565012}，请尽快处理，这是账号邮箱wawzj1255173656@163.com，附上付款记录截图
![微信图片_20241125121321](https://github.com/user-attachments/assets/918464bb-3635-455e-b263-bef7f7d47871)

**Comment by Ginny125** [comment]
![微信图片_20241125121707](https://github.com/user-attachments/assets/30ba8981-0fbb-4e5f-a599-b55cce416f96)
附上蓝灯账号截图，@Mike777Khoo 请尽快处理

**Comment by MaxMKMax** [comment]
@Ginny125 您好. 已经帮您提交问题上去处理了. 谢谢您的耐心等待.

**Reply by Ginny125** [reply]
@Mike777Khoo 蓝灯已经变免费版了，但没有收到退款，我钱呢我钱呢我钱呢

**Reply by Ginny125** [reply]
快点搞定我的退款事宜！真的要说难听的话才肯处理？
